package.json

```diff
@@ -1,10 +1,11 @@
 {
   "scripts": {
-    "dist": "ncc build src/pr-notify.js",
```

Comment:
huh... what was this doing before? It seems like what you have now is more accurate, at least.